docker/prisma-sync.dockerfile

```
@@ -1,5 +1,38 @@
+COPY packages/config/package.json ./packages/config/package.json
+COPY packages/db/package.json ./packages/db/package.json
+COPY packages/logger/package.json ./packages/logger/package.json
+COPY packages/next/package.json ./packages/next/package.json
+COPY packages/ui/package.json ./packages/ui/package.json
+RUN bun install --ignore-scripts
+
+# Sources the bundle needs: the backfill imports @asm/db, which pulls in the
+# whole db package graph.
+COPY scripts ./scripts
+COPY packages ./packages
+RUN cd packages/db && bunx prisma generate
```

Hadolint (2.15.1)

[warning] Use WORKDIR to switch to a directory

(DL3003)